createUserFeatureFlag/main.go

```diff
@@ -83,6 +83,8 @@ func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse,
 		Status:    strings.ToUpper(requestBody.Status),
 		CreatedAt: time.Now().Unix(),
 		CreatedBy: requestBody.UserId,
+		UpdatedAt: time.Now().Unix(),
```

Comment:
was this not being updated earlier? @shubham-y

Reply:
Yes, previously I missed it.